corehq/apps/export/models/new.py

```diff
@@ -452,21 +456,12 @@ def _insert_form_repeat_system_properties(cls, table, columns):
         columns.insert(0, existing_column or ROW_NUMBER_COLUMN)
 
     @classmethod
-    def _insert_parent_case_table(cls, instance):
+    def _insert_parent_case_table(cls, table, columns):
```

Comment:
nit: `_insert_parent_case_table_properties()` might be a better name for the function